alembic/versions/0e105f6e36b4_add_a_commit_column_to_the_builds_table.py

```diff
@@ -0,0 +1,25 @@
+"""Add a commit column to the builds table.
+
+Revision ID: 0e105f6e36b4
+Revises: c6a5e2849ca4
+Create Date: 2017-05-11 18:35:40.550295
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0e105f6e36b4'
+down_revision = 'c6a5e2849ca4'
+
+
+def upgrade():
+    """Add a commit column to the builds table with a uniqueness constraint."""
+    op.add_column('builds', sa.Column('commit', sa.Unicode(length=100), nullable=True))
```

Comment:
+1 for 256, I wonder if MySQL doesn't support varchar up to 255 so we may want to use this instead but either one is fine for me